cx-server-companion/files/init-nexus.js

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/*
+ * Run initialization script on nexus.
+ * The actual script is in `nexus-init-repos.groovy`.
+ */
+
+const request = require('request')
+const fs = require('fs')
+const _ = require('underscore')
+
+if (process.argv.length < 3) {
+    throw new Error("Usage: node init-nexus.js template-values-object [application-configuration-object]")
+}
+
+const templateValues = JSON.parse(process.argv[2])
+let appConfig = {}
+if (!_.isUndefined(process.argv[3])) {
+    appConfig = JSON.parse(process.argv[3])
+}
+
+function verboseLog() {
+    if (appConfig.verbose) {
+        console.log.apply(null, arguments)
+    }
+}
+
+verboseLog(templateValues)
+
+let baseUrl = 'http://cx-nexus:8081/'
```

Comment:
hard coded? Does this apply always?

Reply:
This URL is exposed only to the Jenkins container for caching purpose.

Reply:
Yes, applies always.